tests/security/openscap/oscap_setup.pm

```
@@ -15,14 +15,15 @@ use version_utils 'is_opensuse';
 
 sub run {
 
-    zypper_call('in openscap-utils libxslt-tools');
+    zypper_call('in openscap-utils libxslt-tools patch');
 
     oscap_get_test_file("oval.xml");
     oscap_get_test_file("xccdf.xml");
-
-    # xccdf.xml is for SLE, the CPE is different on openSUSE
     if (is_opensuse) {
-        assert_script_run "sed -i 's#cpe:/o:suse#cpe:/o:opensuse#g' xccdf.xml";
+        oscap_get_test_file("oscap-dict.patch");
+        oscap_get_test_file("oscap-oval.patch");
+        assert_script_run("patch -u -b /usr/share/openscap/cpe/openscap-cpe-dict.xml -i oscap-dict.patch");
+        assert_script_run("patch -u -b /usr/share/openscap/cpe/openscap-cpe-oval.xml -i oscap-oval.patch");
```

Same here, let it fail for leap 15.5 until it is fixed